code/modules/client/preference/loadout/loadout_gloves.dm

```diff
@@ -5,4 +5,12 @@
 
 /datum/gear/gloves/fingerless
 	display_name = "Fingerless Gloves"
-	path = /obj/item/clothing/gloves/fingerless
\ No newline at end of file
+	path = /obj/item/clothing/gloves/fingerless
+
+/datum/gear/gloves/silverring
+	display_name = "Silver ring"
+	path = /obj/item/clothing/gloves/ring/silver
```

Comment:
abuseable

Reply:
How are the rings abuseable?

Reply:
After extensive testing and code checking in coder chat we've been unable to figure out what is abusable about rings. They can't be used to get materials, don't block prints, can't be put into vendors for premium items. So what is wrong with rings?

Reply:
I _think_ since rings are gloves they hide fingerprints, so starting with one of those rings will mean you don't have to worry about getting gloves before committing crimes.

Reply:
I am very sure https://github.com/ParadiseSS13/Paradise/blob/fdc208c3e0be05ed08b1abf57601525e16f118e5/code/modules/clothing/gloves/rings.dm#L1

Reply:
Ah, I think I was thinking of if the gloves let you unscrew hot lightbulbs or something. I had this vague memory of rings doing something that doesn't make sense.

Reply:
Probably fine--I forget what I even opposed this for.

Reply:
OK